spec/beaker/hypervisor/docker_spec.rb

```diff
@@ -280,7 +280,7 @@ module Beaker
           hosts.each do |host|
             expect(docker).not_to receive(:install_ssh_components)
             expect(docker).not_to receive(:fix_ssh)
-            expect(docker).to receive(:dockerfile_for).with(host).and_return('')
+            allow(docker).to receive(:dockerfile_for).with(host).and_return('')
```

Comment:
If you want to make sure it was called, you should at the end add `expect(docker).to have_received(:dockerfile_for).with(host)`.

Reply:
ah, nice catch! let me fix it